runtime/src/main/resources/application.properties

```diff
@@ -6,12 +6,18 @@
 server.port=8080
 server.shutdown=graceful
 spring.lifecycle.timeout-per-shutdown-phase=300s
-spring.profiles.active=zitadel,zitadel-testbed,terraform-boot
+spring.profiles.active=zitadel,zitadel-testbed,nflow.db.h2
```

Comment:
need a simple profile name.

Reply:
@swaroopar Hi Swaroop
![image](https://github.com/eclipse-xpanse/xpanse/assets/122504203/13a37b37-2074-4244-8540-6b432604fe44)
According to the definition of nFlow, we need to activate its H2 or mariadb data source. The activation key of the H2 database is nflow.db.h2, which we cannot modify.

Reply:
@WangLiNaruto ok got it. then we can keep it as it is.